### PR TITLE
Test URL Alias editing and how that affects the static build.

### PR DESCRIPTION
**Pull request by jonpugh**
- Adds rm -rf /docroot/static
- Test URL alias editing and  how that affects the static build.

### CODE REVIEW

File: tests/behat/features/decoupled.feature [hunk 1]
```diff
@@ -33,12 +33,33 @@ Feature: The VA Website is generated inside the Drupal CMS code.
...
     Then I should see "VA blind and low vision rehabilitation services - EDITED"
     And I am at "/static/health-care/about-va-health-benefits/vision-care/blind-low-vision-rehab-services"
     Then I should see "VA blind and low vision rehabilitation services - EDITED"
     Then print current URL
+
+    Given I am at "/static/disability/effective-date/edited"
+    Then the response status code should be 200
+    And the "a:empty" element should exist
+
+    Given I am at "/static/disability/about-disability-ratings/effective-date"
```
Comment (swirtSJW) [inline tests/behat/features/decoupled.feature]: My understanding is that the super team is defining areas of concern as  front-end, content, and cms.   This test is blocking cms based on front-end failures.   I think this entanglement of concerns should be avoided.

Just today we had 5 smart CMS and front-end people spend hours tracking down a content issue.   We should be avoiding that.

File: tests/behat/features/decoupled.feature [hunk 2]
```diff
@@ -33,12 +33,33 @@ Feature: The VA Website is generated inside the Drupal CMS code.
...
+    # @TODO: Old URLs should redirect to the latest URL.
+    # This page should NOT Exist in HTML, it should redirect.
+    Then the response status code should not be 200
```
Comment (swirtSJW) [inline tests/behat/features/decoupled.feature]: This is a test of the front-end's implementation of redirects, not the CMS.  The entangling of concerns should be remediated by testing only that the redirect shows up in the qraphQL that CMS provides the front-end.  It is up to the front-end team to test that they are building redirects appropriately and in a  timely fashion from that graphQL endpoint.

File: tests/behat/features/decoupled.feature [hunk 3]
```diff
@@ -33,12 +33,33 @@ Feature: The VA Website is generated inside the Drupal CMS code.
...
     When I run "cd .. && composer va:web:build"
     And I am at "/health-care/about-va-health-benefits/vision-care/blind-low-vision-rehab-services"
     Then I should see "VA blind and low vision rehabilitation services - EDITED"
     And I am at "/static/health-care/about-va-health-benefits/vision-care/blind-low-vision-rehab-services"
     Then I should see "VA blind and low vision rehabilitation services - EDITED"
     Then print current URL
+
+    Given I am at "/static/disability/effective-date/edited"
+    Then the response status code should be 200
+    And the "a:empty" element should exist
```
Comment (swirtSJW) [inline tests/behat/features/decoupled.feature]: Seems like a bit of a loose test.  Why are we checking for the existence of any empty a?

File: tests/behat/features/decoupled.feature [hunk 4]
```diff
@@ -33,12 +33,33 @@ Feature: The VA Website is generated inside the Drupal CMS code.
...
+
+    Given I am at "/static/disability/about-disability-ratings/effective-date"
+    # @TODO: Old URLs should redirect to the latest URL.
+    # This page should NOT Exist in HTML, it should redirect.
+    Then the response status code should not be 200
+
+    # @TODO: Empty A tags should not appear, but these pages shouldn't exist either, so this test might not really be needed.
+    And the "a:empty" element should exist
```
Comment (swirtSJW) [inline tests/behat/features/decoupled.feature]: We addressed this a while ago with the front-end team.  There are allowable instances of empty a tags.  The front-end team has already adjusted their tests for this (which will become a content test in the content area of concern)   We should not be testing  it here on an existing page that is prone to being edited.

**Comment by jonpugh** [comment]
This should be fine.

The code simply replaces the $environment "name" with "name.web".

https://github.com/department-of-veterans-affairs/devops/blob/ansible/build/cms-ci/ansible/build/roles/cms-ci/va_gov_devshop/va_gov_devshop.module#L699

```php
/**
 * Implements hook_devshop_environment_alter().
 */
function va_gov_devshop_devshop_environment_alter(&$environment, $project) {
  if ($project->github_owner == "department-of-veterans-affairs" && $project->github_repo == "va.gov-cms") {
    $environment->uri_web = str_replace($environment->name, $environment->name . '.web', $environment->uri);
    $environment->url_web = 'http://' . $environment->uri_web;

    // Add the WEB uri to the domains list.
    $environment->domains[] = $environment->uri_web;

    // @TODO: Handle forks.
    $environment->github_owner_web = $project->github_owner;
    $environment->github_repo_web = "vets-website";
  }
}
```

**Comment by swirtSJW** [comment]
Closing this because of it being so outdated.
There is a question of whether we need this in a separate pull request.
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/643/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R351

**Comment by swirtSJW** [comment]
This got re-opened by mistake.

**Pull request closed without merging**